spinalcordtoolbox/straightening.py

```diff
@@ -374,22 +374,15 @@ def straighten(self):
             plt.savefig('fig_straighten_' + datetime.now().strftime("%y%m%d%H%M%S%f") + '.png')
             plt.close()
 
-        # alignment_mode = 'length'
-        alignment_mode = 'levels'
```

Comment:
I'm curious about the commented-out `alignment_mode = 'length'`. I'm wondering when that happened, and if there were ever plans for this to be an option somehow.

Maybe it was one of those debugging/development-only comments where it would be uncommented on the fly during research to test some different result?

Nonetheless, I agree that it's currently useless to our stable-release userbase.

Reply:
That's my guess as well.